src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -1,5 +1,7 @@
 package nl.tudelft.pixelperfect;
 
+import java.io.IOException;
+
```

Comment:
Thanks. Is this error reported by your local plugin, or by the Maven reporting process? (If local, please refer to the discussion on this matter in the following PR: https://github.com/jessetilro/pixelperfect/pull/116)